server/src/controllers/eventController.ts

```diff
@@ -10,12 +10,21 @@ import {
   getDiscountAmount,
   updateContact,
   validateDiscount,
+  validateWithRegex,
 } from './eventController.service';
 import {updateCanceledOrder, orderFulfillment} from './orderController.service';
 import {extendPrismaClient} from './PrismaClient/GetExtendedPrismaClient';
 import {isBooleanString} from 'class-validator';
 const prisma = extendPrismaClient();
 
+import multer from 'multer';
+import {Storage} from '@google-cloud/storage';
+
+const upload = multer();
+
+const storage = new Storage({keyFilename: './wondertix-app-12f15702b745.json'});
```

Comment:
This should be an environment variable and we'll want to point to a different bucket for dev, staging, and prod. Ideally we don't have to use actually cloud storage for locally running environments. That would mean starting up a `fake` cloud storage image in docker that the locally running server talks to. But given the time frame, and given that we already use external services for local development (auth0), an acceptable solution would be to have locally running environments point to another shared bucket in cloud storage (eg. `image_upload_wondertix_local`).

Reply:
Here's another PR that adds a new environment variable for reference https://github.com/WonderTix/WonderTix/pull/528

Reply:
The bucket name and bucket key file now use an env variable